cli/dcoscli/job/main.py

```diff
@@ -588,8 +591,11 @@ def _add_job(job_file):
         _post_job(full_json)
         job_added = True
     except DCOSHTTPException as e:
-        if e.response.status_code == 409:
-            emitter.publish("Job ID: '{}' already exists".format(job_id))
+        if e.response.status_code == 422:
```

Comment:
Since we are no longer having a custom message here, we should remove the `try/except`. we catch `DCOSHTTPException` in `http.request`.